capstone/backend/src/main/java/com/google/sps/servlets/CreateOrUpdateUserlistServlet.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps.servlets;
```

Comment:
do you mind moving this servlet to `java/com/google/servlets` directory? So the package is `com.google.servlets`. (delete the sps directory, since we don't need it).

Reply:
Yeah, I will move them.